packages/neo4j-driver/examples/neo4j.html

```diff
@@ -265,7 +265,7 @@ <h1>
 
       if (document.location.href.indexOf('example') >= 0) {
         document.getElementById('statement').textContent =
-          'MERGE (alice:Person {name: $name_a, age: $age_a})\nMERGE (bob:Person {name: $name_b, age: $age_b})\nCREATE UNIQUE (alice)-[alice_knows_bob:KNOWS]->(bob)\nRETURN alice, bob, alice_knows_bob'
+          'MATCH (alice:Person {name: $name_a, age: $age_a})\nMATCH (bob:Person {name: $name_b, age: $age_b})\nMERGE (alice)-[alice_knows_bob:KNOWS]->(bob)\nRETURN alice, bob, alice_knows_bob'
```

Comment:
```suggestion
          'MERGE (alice:Person {name: $name_a, age: $age_a})\nMERGE (bob:Person {name: $name_b, age: $age_b})\nMERGE (alice)-[alice_knows_bob:KNOWS]->(bob)\nRETURN alice, bob, alice_knows_bob'
```

`alice` and `bob` might don't exist, so we should use `MERGE` instead of `MATCH` in this case.

Reply:
I'd go further and set the age of each person only in the `MERGE`'s `ON CREATE` clause